paperpress.js

```diff
@@ -117,13 +118,40 @@ Paperpress.prototype._fileToItem = function (file) {
 
 	var item = {
 		title: name,
-		slug: slug
+		slug: slug,
+		content: fileContent,
+		path: null
 	}
 
-	item.path = this.pathBuilder(item, file.collectionName)
-	item.content = (fileType === '.md') ? this._marked.render(fileContent) : fileContent
+	if (frontMatter.test(fileContent)) {
+		var contentData = frontMatter(fileContent)
+		var dataAttributes = contentData.attributes
+
+		// Iterate through attributes and merge into the default item
+		for (var k in dataAttributes) {
+			if (dataAttributes.hasOwnProperty(k)) {
```

Comment:
`dataAttributes.hasOwnProperty(k)` is correct. Checks against it self to prevent attrs leaking from a proto change on object. If done against item it checks item proto.